deploy/helm/values-uat.yaml

```diff
@@ -13,10 +13,39 @@ resources:
     memory: 125Mi
 
 ingress:
+  enabled: true
   className: default-non-prod
   annotations:
     external-dns.alpha.kubernetes.io/aws-weight: "100"
-    ingressClassName: nginx
+    nginx.ingress.kubernetes.io/server-snippet: |
+      if ($scheme = http) {
+        return 444;
+      }
+
+      # Block requests on weekends
+      if ($date_gmt ~* "^(Sat|Sun)") {
+        return 503;
+      }
+
+      # Block requests between 21:00 and 05:59 UTC (22:00 and 06:59 BST)
+      # if ($time_local ~ ":(2[1-3]|0[0-5]):") {
+      if ($time_local ~ ":(2[1-3]|[00-12]):") {
```

Comment:
this looks to be returning 503 between 21:59 and 12:59 UTC  - should it not be 05:59 UTC for the latter? 

I think the `[00-05]` would not work as well as it is a string. Does the original `":(2[1-3]|0[0-5]):")` not work?